2024/half-2/day-15/size/size.ts

```
@@ -0,0 +1,32 @@
+export default function size<
+  T extends
+    | Array<any>
+    | Object
```

Biome

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)